### ISSUE TITLE
Collaboration

### ISSUE THREAD

**Issue by xloem** [datacoinproject/datacoin#1]
What work is still needed on this?  Is it stable?

What do visitors need to know when they find this?

**Comment by xloem** [comment]
@gjhiggins should PRs go in your repository or this one?  (if yours is a replacement could you turn issues on?)

**Comment by gjhiggins** [comment]
Thanks for your contributions and your interest. I've enabled issues for my repos. To answer your question - this particular repos is in limbo atm, subscribers to the bitcointalk thread have reported various issues with txs between the 0.8 client and this 0.16.3 client so for the time being, I've continued development in my own repos, marking it testnet-only until we're all a bit more confident about its reliablity.

**Comment by xloem** [comment]
thank you

**Comment by xloem** [comment]
are you able to reproduce any of these bugs?  I checked the thread and made a post as baffo32, but it's pretty long to figure out which bugs are still active.  If they're reproducible it's pretty easy to use `git bisect` to hunt down what started them.

**Comment by gjhiggins** [comment]
No, I can't reproduce the reported issues. `senddata` is working as expected (on both 0.8 and 0.16.3 clients on testnet) nor can I reproduce either of:
- broadcast tx from 0.8 client  it is not propagated across 0.16 nodes in the network
- broadcast tx from 0.16 client the tx is never broadcast nor confirmed”
I've successfully sent txs back and forth between 0.8 and 0.16.3 clients on testnet.

**Comment by xloem** [comment]
Who is complaining of the issue?  Maybe what might be next would be to verify a client can sync with any 0.8 peers still on the network.

**Comment by gjhiggins** [comment]
Coupla people on the bitcointalk thread. Excellent suggestion, tried it, revealed a mistranscription which inappropriately excluded headers from 0.8 clients (meaning 0.8 clients could sync from 0.16 but not vice versa). I corrected the transcription and my 0.16 client is now busily syncing from the one and and only listening 0.8(.3) client (151.80.96.105) - https://github.com/gjhiggins/datacoin/commit/f14baf965829ba33abbb06b41b768fe5e42dea24

**Comment by gjhiggins** [comment]
btw, there's a Datacoin Discord server https://discord.gg/evxVsb5FXB

**Comment by gjhiggins** [comment]
Good catch. The recompiled 0.16.3 client is fully synced, took a couple of hours to sync from 0.
